core/templates/pages/thanks-page/thanks-page.component.spec.ts

```diff
@@ -16,29 +16,32 @@
  * @fileoverview Unit tests for thanks page component.
  */
 
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// App.ts is upgraded to Angular 8.
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
-require('pages/thanks-page/thanks-page.component.ts');
+import { ThanksPageComponent } from './thanks-page.component';
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service.ts';
```

Comment:
Is this import required? I think it is not required. Can you check once?

Reply:
Done